nixos/modules/services/continuous-integration/github-runner/options.nix

```diff
@@ -74,6 +74,16 @@ with lib;
     default = null;
   });
 
+  rawNameSuffix = mkOption {
```

Comment:
```suggestion
  nameSuffix = mkOption {
```

Minor bikeshed.

Reply:
I ended up just using `name` and removing the `escapeShellArg`, so removed this option